tikv/raftstore/applier.go

```diff
@@ -434,7 +433,11 @@ func notifyStaleReq(term uint64, cb *Callback) {
 }
 
 /// Checks if a write is needed to be issued before handling the command.
-func shouldWriteToEngine(cmd *raft_cmdpb.RaftCmdRequest, wbKeys int) bool {
+func shouldWriteToEngine(rlog raftlog.RaftLog, wbKeys int) bool {
+	cmd := rlog.GetRaftCmdRequest()
+	if cmd == nil {
+		return false
```

Comment:
Is checking wbKeys >= WriteBatchMaxKeys not required when using new raft log?

Reply:
The check is not required, it's based on benchmarks of RocksDB.
We probably gonna remove it totally.